src/components/ui/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import Accordion from './Accordion';
+
+export default {
+  title: 'Ui/Accordion',
+  component: Accordion,
+  argTypes: {},
+} as ComponentMeta<typeof Accordion>;
+
+const Template: ComponentStory<typeof Accordion> = (args) => (
+  <Accordion {...args}>
+    <div>Click me</div>
```

Comment:
This story doesn't work
`Accordion` should be used in conjunction with `AccordionDetails` and `AccordionSummary` so let's demonstrate it